apps/discord/commands.py

```
@@ -0,0 +1,108 @@
+from typing import Dict, List
+
+import aiohttp
+from config import ALLOWED_GUILD_IDS, DiscordResponseType
+from discord_client import DiscordClient
+from link_shortener import LinkShortener
+
+
+class CommandHandler:
+    """Handles Discord slash commands."""
+
+    def __init__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)